docs/data/sdks/browser-2/migration.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Browser SDK 2.0 Migration Guide
+description: Use this guide to easily migrate from Amplitude Browser SDK 1.0 to the Browser SDK 2.0.
+---
+
+Amplitude Browser SDK 2.0 (`@amplitude/analytics-browser`) features default event tracking, improved marketing attribution tracking, simplified interface and a lighter weight package.
+
+### Terminology
+
+* `@amplitude/analytics-browser@1`: Browser SDK 1.0
+* `@amplitude/analytics-browser@2`: Browser SDK 2.0
+
+### Dependency
+
+For snippet installation, update your project's [snippet loader](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader).
+
+For Node projects, update your dependency list in package.json.
+
+```diff
+{
+  "dependencies": {
+-     "@amplitude/analytics-browser": "^1"
++     "@amplitude/analytics-browser": "^2"
+  }
+}
+```
+
```

Comment:
Is the `userID` `config` change still included? We should call out the ability to pass in config as a second parameter.

Reply:
it is included and i can update to w/o user id :)